tests/compat.py

```diff
@@ -34,3 +34,7 @@
     from io import TextIOWrapper
 except ImportError:  # Python < 3
     TextIOWrapper = file
```

Comment:
@jlaska Yep, noqa was what I was grasping for. Edited things to use that approach with recent commit. I'll give it a more thorough review later, but that plugs all the major holes.